ProcessLib/HeatTransport/CreateHeatTransportProcess.cpp

```diff
@@ -32,52 +32,53 @@ std::unique_ptr<Process> createHeatTransportProcess(
     // Process variable.
     auto process_variables = findProcessVariables(
         variables, config,
-        {//! \ogs_file_param_special{process__HEAT_TRANSPORT__process_variables__process_variable}
+        {//!
+         //\ogs_file_param_special{process__HEAT_TRANSPORT__process_variables__process_variable}
```

Comment:
These comments must not be broken up into two lines. Otherwise Doxygen won't recognize them later on.